spec/support/shared_examples/graphql_requirements.rb

```diff
@@ -20,6 +20,18 @@
   end
 end
 
+RSpec.shared_examples "requires Premium license" do
```

Comment:
Since the two shared sexamples have almost the same title, I feel like this could be very confusing. I'd suggest considering renaming them:

```suggestion
RSpec.shared_examples "a resolver which requires Premium license" do
```